File: java/src/main/java/algorithm/W.java
package algorithm;

public class W implements Data {

    public final int dy;
    public final int mxT;
    public final int mnT;

    public W(int dy, int mxT, int mnT) {
        this.dy = dy;
        this.mxT = mxT;
        this.mnT = mnT;
    }

    @Override
    public String r() {
        return Integer.toString(dy);
    }

    @Override
    public int sort() {
        return mxT - mnT;
    }

}

File: javascript/src/w.js
class W {

    constructor(dy, mxT, mnT) {
        this.dy = dy;
        this.mxT = mxT;
        this.mnT = mnT;
    }

    r() {
        return "" + this.dy;
    }

    sort() {
        return this.mxT - this.mnT;
    }
}

module.exports = {
    W
};
